apps/miiverse-api/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:20-alpine
+
+RUN apk add --no-cache python3 make gcc g++ 
```

Comment:
What's the need for stuff like Python here? The older Dockerfile didn't have this?

Reply:
Needed for node-gyp. Any npm dependency that requires compiling (rust, c, cpp, zig) needs these dependencies.

Reply:
I phrased this very poorly, that's my bad for doing review so late at night. My apologies

I know what they're used for, the main question I was trying to get at was why it was explicitly being installed here, whereas the original Dockerfile didn't. I'm unsure what changed that now requires a manual installation of these dependencies

Reply:
Ah I see the problem, I originally used the master branch for both projects. The master branch held the dockerfile you're seeing.

I later used the dev branch to replace the source code files, public files and package.json files. but apparently forgot the dockerfile.

I'll update the dockerfile when I come home from work today. (dont let that stop your rereview though)

Reply:
Dockerfiles have been updated

Reply:
Gotcha okay, that makes sense then 👍